### ISSUE TITLE
Score Bug 

### ISSUE THREAD

**Issue by andrefarrugia96** [rose2hnd1i/IMTAssignment2016#6]
Whilst playing the game I could not understand the functionality of the score. Is it a bug ?

**Comment by rose2hnd1i** [comment]
Yes, I'm having a problem to update the score when the character picks up the coin. I will fix this issue in the next commit.
